pokesummary/displaying.py

```diff
@@ -147,3 +147,22 @@ def display_summary(pokemon_name, pokemon_stats):
     type_defenses = calculate_type_defenses(pokemon_stats)
     print(get_type_defenses_chart(type_defenses))
     print()
+
+    #show pokemon pre evolution
+    while(pokemon_stats["pre_evo_id"]):
+        pre_evo = display_pre_evo(pokemon_stats["pre_evo_id"])
+        pre_evo_name = pre_evo["pokemon_name"]
+        pre_evo_details = pokemon_stats["evo_details"]
+        print(f"{pre_evo_name} evolves into {pokemon_name} at {pre_evo_details or 'trade'}")
+        pokemon_stats=pre_evo
+        pokemon_name=pre_evo_name
+
+
+def display_pre_evo(pre_evo):
+    data_dictionary = parsing.csv_to_nested_dict(
+        "pokesummary.data",
+        "pokemon_modified_new.csv",
+        "pokemon_id"
+    )
```

Comment:
This is a little clunky; it seems unnecessary to read in the entire CSV again for every pre-evolution.

Reply:
i know, but that was the only way i was able to find at that moment. i just did all of the code as fast as i could without fearing it would make it a spagetthi code 😅. but all of the code works i checked